### DISCUSSION TITLE
Unexpected mouse cursor (arrow instead of I-beam) when selecting text in the terminal area on macOS

### DISCUSSION THREAD

**Discussion by w0rm** [ghostty-org/ghostty#3565] [Feature Requests, Ideas] [answered]
Hi and congratulations on the release!

I've been using macOS Terminal before, and having tried Ghostty, I noticed that the mouse cursor is an arrow instead of the I-beam used for selecting text. It would feel more native if it was an I-beam.

![ghostty](https://github.com/user-attachments/assets/a1600347-d779-4394-8a38-0842136da5b7)

**Comment by yawaramin** [comment]
Cross ref https://github.com/ghostty-org/ghostty/discussions/3257

**Comment by 00-kat** [comment]
Cross-referencing #5384, not the same issue but very similar.

**Comment by mitchellh** [comment, marked as answer]
#8409
